sormas-api/src/main/resources/strings.properties

```diff
@@ -921,3 +921,14 @@ SurvnetGateway.notificationEntryNotSent = Entry could not be sent
 SurvnetGateway.notificationErrorSending = Error when sending entry
 
 warningDashboardMapTooManyMarkers = There are %d places to display and it is possible that your browser will freeze while displaying them.
+
+# Messaging
+sms = SMS
```

Comment:
Most of the additions here should be moved to captions because they're used as captions. enterSMS is probably the only one that should stay here.